docs/user/docs/sessions/files/files.md

```diff
@@ -42,4 +42,29 @@
 
 # Download
 
-The documentation for the download feature will follow soon.
+<!-- prettier-ignore-start -->
+
+1. Navigate to `Sessions`
+1. On the right side, select the corresponding session. Select the persistent
+   session to upload files into your persistent workspace. Click the
+   `File Browser`:
+
+    ![File Browser button](file-browser-button.png)
+
+1. Wait until the list of files has loaded.
+
+   ![Download button](download-button.png)
+
+!!! info
+    To download files from hidden directories, e.g., the workspace
+    metadata, one has to expand these directories by clicking
+    `Show hidden files`
+
+1. Expand the directories by clicking on the directory icon (1) until you find the directories where you want to
```

Comment:
Those lines are a copy of what's written for the upload section. 
I think the docs can be organized a bit smarter, so we can get rid of this duplication.

Reply:
I think only a few of them are and was not sure how a user would go into reading it so I would leave it for now